src/pages/Login/index.tsx

```diff
@@ -85,89 +78,124 @@ const LoginPage: React.FC = () => {
   /**
    * Display login form to user
    */
-  const renderForm = () => (
-    <MaxWidthBox maxWidth={'500px'} paddingLeft={'50px'} paddingRight={'50px'}>
+  const renderLoginForm = () => (
```

Comment:
Does this function need to be defined within the render function? I would believe that is a performance hit if we are redefining this function on each call.

Reply:
True, I'll move LoginForm to be its own stateless component